### PR TITLE
Cors update

### LINKED ISSUE #1735: security/cors.rst "Enabling Cross-Origin Requests (CORS)"
- [x] Update for RTM

### PR DESCRIPTION
**Pull request by spboyer**
Closes #1735 

- All Code samples updated

### CODE REVIEW

**Comment by danroth27** [comment]
Let's not use "RTM" in the code. There will be many RTM releases of ASP.NET Core and we should make the code as timeless as possible.

File: aspnet/security/cors/sample/src/CorsExamplesRTM/CorsExample2/Project_Readme.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
+﻿<!DOCTYPE html>
```
Comment (danroth27) [inline aspnet/security/cors/sample/src/CorsExamplesRTM/CorsExample2/Project_Readme.html]: Remove this file

**Comment by spboyer** [comment]
ok @danroth27 looks like all good now.

File: aspnet/security/cors.rst [hunk 2]
```diff
@@ -37,16 +36,17 @@ Setting up CORS
...
 To setup CORS for your application you use the ``Microsoft.AspNetCore.Cors`` package. In your project.json file, add the following:
 
-.. literalinclude:: cors/sample/src/CorsExample1/project.json
-  :language: none
-  :lines: 5,6,9
-  :emphasize-lines: 2
+.. literalinclude:: cors/sample/src/CorsExamples/CorsExample1/project.json
+  :language: javascript
+  :lines: 2-12
+  :emphasize-lines: 10
```
Comment (Rick-Anderson) [inline aspnet/security/cors.rst]: This is an advanced sample, should we think about just telling them what line to add **AND** not even have the version number?

**Comment by spboyer** [comment]
@danroth27 Thoughts on this ^^ - this update is an update on the code samples. Being a reference doc, not sure including or excluding this information makes any difference.

**Comment by danroth27** [comment]
It would be good to avoid tying this doc to a specific package version number if we can avoid it. Other than that :shipit:.

**Pull request merged**